modules/admin_manual/pages/release_notes.adoc

```diff
@@ -40,6 +40,11 @@ Version 10.1 delivers all the prerequisites to be compatible with the Microsoft
 
 ownCloud Server 10.1 introduces WebDAV Locks that allow clients to lock and unlock resources to prevent other users from making changes. The feature has been implemented as a prerequisite for manual file locking and MS Office Online Server compatibility. In the current state, file locking is only available via API. Users can recognize locked files via the "lock" icon in the file list. Additionally a lock owner (the user who locked the file) can manually unlock them via the "Locks" tab in the right sidebar. The "Locks" tab will only appear for files that have active locks.
 
+=== Foreign Keys in Database
+
+Please note that foreign keys have been added with the :xref:webdav-locks[WebDAV Locks] feature.
```

Comment:
I have searched all the content of the modules directory but I can´t find any `webdav-locks` reference?
Did I missed something?

Reply:
it's the section just above this one

Reply:
sorry, you are right, overseen how links are created without explicit writing them :man_facepalming:

Reply:
@mmattel, [here's a link](https://asciidoctor.org/docs/user-manual/#automatic-anchors) to the relevant documentation.